services/web/server/src/simcore_service_webserver/cli.py

```diff
@@ -68,7 +68,8 @@ async def app_factory() -> web.Application:
     assert app_settings.SC_BUILD_TARGET  # nosec
 
     _logger.info(
-        "Application settings: %s", app_settings.json(indent=2, sort_keys=True)
+        "Application settings: %s",
+        app_settings.model_dump_json(indent=2),
```

Comment:
MINOR: do I see we miss here the sort_keys ... What about using instead our own `json_dumps`, i.e.
```python
from models_library.utils.json_serialization import json_dumps

_logger.info( "Application settings: %s", json_dumps(app_settings, indent=2, sort_keys=True))
```